src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.Stream.cs

```diff
@@ -333,6 +339,7 @@ public static partial class JsonSerializer
 
         [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
                 Justification = "Workaround for https://github.com/mono/linker/issues/1416. All usages are marked as unsafe.")]
+        [UnconditionalSuppressMessage("AOT", "IL3050:Calling members annotated with 'RequiresDynamicCodeAttribute' may break functionality when AOT compiling.", Justification = "<Pending>")]
```

Comment:
1. The strings in the second string, `IL3050` are really long. Can they just be simplified to `IL3050:RequiresDynamicCodeAttribute".
2. All justification should be filled out.
3. Is the justification the same as the existing suppress message - working around https://github.com/mono/linker/issues/1416? That doesn't seem right. Maybe this method should just have `[RequiresDynamicCode]` on it.